server/controllers/utils/getFormValues.ts

```diff
@@ -112,7 +112,7 @@ export const getFormValues = ({ errors = {}, unsavedValues = {}, apiValues }: Ge
   // missing documents detail
   values.missingDocumentsDetail = isDefined(errors.missingDocumentsDetail)
     ? ''
-    : unsavedValues.missingDocumentsDetail || apiValues.missingDocumentsRecords?.[0]?.details
+    : unsavedValues.missingDocumentsDetail || undefined
```

Comment:
why is this here rather than in the above block with the other 'standard' fields?

Reply:
It's a distinct case, there's no intention of using the API value, which is what the other clauses do

Reply:
I see, because we can never edit an MDR? Should the note detail not also be here? Rescind detail? And other unchangeable fields?

Reply:
Notes, rescind records and missing doc records now handled the same way